### ISSUE TITLE
(HOLD) Remove GA custom dimension - preferred times

### ISSUE THREAD

**Issue by laurenernest** [department-of-veterans-affairs/va.gov-team#29271]
## Description
Request VSP Analytics team remove the custom dimension configured in GA for the contact info preferred times to call parameter in the submission event.

There are limits to the total amount of custom dimensions allowed in GA. We should request this be removed once our analysis is complete to clear this spot up.

**Comment by ldelacosta** [comment]
No longer needed.
